scripts/dataset_processing/spoken_wikipedia/run.sh

```diff
@@ -0,0 +1,112 @@
+tar -xvf en-with-audio.tar
+
+##  We get a folder English with 1339 subfolders, each subfolder corresponds to a Wikipedia article. Example:
+##  ├── Universal_suffrage
+##  │   ├── aligned.swc
+##  │   ├── audiometa.txt
+##  │   ├── audio.ogg
+##  │   ├── info.json
+##  │   ├── wiki.html
+##  │   ├── wiki.txt
+##  │   └── wiki.xml
+
+##  We will use two files: audio.ogg and wiki.txt
+
+## Some folders have multiple .ogg files, this will be handled during preprocess.py. Example:
+##  |── Universe
+##  │   ├── aligned.swc
+##  │   ├── audio1.ogg
+##  │   ├── audio2.ogg
+##  │   ├── audio3.ogg
+##  │   ├── audio4.ogg
+##  │   ├── audiometa.txt
+##  │   ├── info.json
+##  │   ├── wiki.html
+##  │   ├── wiki.txt
+##  │   └── wiki.xml
+
+## Some rare folders are incomplete, these will be skipped during preprocessing.
+
+## path to NeMo repository, e.g. /home/user/nemo
+NEMO_PATH=
+
+INPUT_DIR="english"
+OUTPUT_DIR=${INPUT_DIR}_result
+
+rm -rf $OUTPUT_DIR
+rm -rf ${INPUT_DIR}_prepared
+mkdir ${INPUT_DIR}_prepared
+mkdir ${INPUT_DIR}_prepared/audio
+mkdir ${INPUT_DIR}_prepared/text
+python ${NEMO_PATH}/scripts/dataset_processing/spoken_wikipedia/preprocess.py --input_folder ${INPUT_DIR} --destination_folder ${INPUT_DIR}_prepared
+
+## Now we have ${INPUT_DIR}_prepared folder with the following structure:
+##  ├── audio
+##  |   ├── 1.wav
+##  |   ├── 2.wav
+##  |   ...
+##  └── text
+##      ├── 1.txt     
+##      ├── 2.txt     
+##      ...
+
+MODEL_FOR_SEGMENTATION="QuartzNet15x5Base-En" 
+MODEL_FOR_RECOGNITION="stt_en_conformer_ctc_large"
+## We set this threshold as very permissive, later we will use other metrics for filtering
+THRESHOLD=-10
+
+${NEMO_PATH}/tools/ctc_segmentation/run_segmentation.sh \
+--SCRIPTS_DIR=${NEMO_PATH}/tools/ctc_segmentation/scripts \
+--MODEL_NAME_OR_PATH=${MODEL_FOR_SEGMENTATION} \
+--DATA_DIR=${INPUT_DIR}_prepared \
+--OUTPUT_DIR=${OUTPUT_DIR} \
+--MIN_SCORE=${MIN_SCORE}
+
+# Thresholds for filtering
+CER_THRESHOLD=20
+WER_THRESHOLD=30
+CER_EDGE_THRESHOLD=30
+LEN_DIFF_RATIO_THRESHOLD=0.15
+EDGE_LEN=25
+BATCH_SIZE=1
+
+${NEMO_PATH}/tools/ctc_segmentation/run_filter.sh \
+--SCRIPTS_DIR=${NEMO_PATH}/tools/ctc_segmentation/scripts \
+--MODEL_NAME_OR_PATH=${MODEL_FOR_RECOGNITION} \
+--BATCH_SIZE=${BATCH_SIZE} \
+--MANIFEST=$OUTPUT_DIR/manifests/manifest.json \
+--INPUT_AUDIO_DIR=${INPUT_DIR}_prepared/audio/ \
+--EDGE_LEN=${EDGE_LEN} \
+--CER_THRESHOLD=${CER_THRESHOLD} \
+--WER_THRESHOLD=${WER_THRESHOLD} \
+--CER_EDGE_THRESHOLD=${CER_EDGE_THRESHOLD} \
+--LEN_DIFF_RATIO_THRESHOLD=${LEN_DIFF_RATIO_THRESHOLD}
+
+python ${NEMO_PATH}/examples/asr/speech_to_text_eval.py \
+  dataset_manifest=${OUTPUT_DIR}/manifests/manifest_transcribed_metrics_filtered.json \
+  use_cer=True \
+  only_score_manifest=True
+
+python ${NEMO_PATH}/examples/asr/speech_to_text_eval.py \
+  dataset_manifest=${OUTPUT_DIR}/manifests/manifest_transcribed_metrics_filtered.json \
```

Comment:
i thought we had cer/wer after the filtering step, no?

Reply:
I do not see this information in the output.
I mean the WER/CER of the whole dataset, not each sentence.